tests/pretrained_test.py

```diff
@@ -6,6 +8,10 @@
 from allennlp_models import pretrained
 
 
+# But default we don't run these tests
+@pytest.mark.skipif(
+    not os.environ.get("ALLENNLP_MODELS_RUN_PRETRAINED_TEST"), reason="requires massive downloads"
+)
```

Comment:
The GPU tests are selected in a different way, aren't they? Why isn't this done the same way?

Reply:
Oh yea, we should probably run those on the self-hosted runner as well

Reply:
Sorry - I realize your question now. I felt like there be an explicit skip condition here because if you were running tests locally you might not want to download these massive files, and I didn't know what that condition could be other than an environment variable flag

Reply:
But we could also add a custom mark like we do for GPU tests and then exclude those by default. That might be cleaner

Reply:
Looks like we're inconsistent about how we handle GPU tests. Some have `@pytest.mark.gpu`, others use `skipif` and detect how many GPUs there are. Others yet have both.

Reply:
The `skipif` ensures they're never accidentally run if you don't have a GPU available. The `mark.gpu` is just so that we can select only GPU tests to run